### PR TITLE
Unit testing

### PR DESCRIPTION
**Pull request by DanielGardiner**

### CODE REVIEW

File: add_item.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            <p>Incubation usual</p>
+            <p><input type="text" name="incubation_usual"></p>
+            <p>Incubation range</p>
+            <p><input type="text" name="incubation_range"></p>
+            <p>Symptoms</p>
+            <p><input type="text" name="symptoms"></p>
+            <p>Severity</p>
+            <p><input type="text" name="severity"></p>
+            <p>Average annual incidence</p>
+            <p><input type="text" name="avg_annual_incidence"></p>
```
Comment (georginaw) [inline add_item.php:34]: This might need to be an input type number? The db will only accept an integer.

File: add_item.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            <p>Image</p>
+            <p><input type="file" name="img_location"></p>
+            <input type="submit">
+        </form>
+    </div>
+
+
```
Comment (georginaw) [inline add_item.php:41]: Delete extra line?

File: add_item.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            <p>Severity</p>
+            <p><input type="text" name="severity"></p>
+            <p>Average annual incidence</p>
+            <p><input type="text" name="avg_annual_incidence"></p>
+            <p>Image</p>
+            <p><input type="file" name="img_location"></p>
+            <input type="submit">
+        </form>
+    </div>
+
```
Comment (georginaw) [inline add_item.php:40]: Delete extra line?

File: add_item.php [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+?>
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Add item</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
+</head>
+<body>
```
Comment (georginaw) [inline add_item.php:11]: Is there a reason that everything is wrapped in the div (class =page) and not just in the body tag? The div and body look to be the same thing.

File: test/functions.php [hunk 5]
```diff
@@ -1 +1,17 @@
 <?php
+require('../functions.php');
```
Comment (georginaw) [inline test/functions.php:2]: Tested using cmd line, all good.

You have the retrieveData function too - do you need to add this to the unit test file?

**Review by georginaw — COMMENTED** [review]
All reviewed, please see comments. Looks great!!

File: test/functions.php (hunk 5 above)
Comment (DanielGardiner) [reply test/functions.php:2]: Can't unit test a function which interacts with a database

File: add_item.php (hunk 4 above)
Comment (DanielGardiner) [reply add_item.php:11]: removed

**Pull request merged**